test/e2e_node/memory_eviction_test.go

```diff
@@ -37,7 +37,9 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
 	f := framework.NewDefaultFramework("eviction-test")
 
 	Context("When there is memory pressure", func() {
-		It("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
+		// Skip the test temporarily because of #30550.
+		// TODO(random-liu): Re-enable this test after #30550 is fixed.
+		PIt("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
```

Comment:
Why is #30476 not acceptible? Disabling a test is not a good idea. This
test has already surfaced two major issues.

On Mon, Aug 15, 2016 at 4:28 PM, Michael Taufen <notifications@github.com>
wrote:

> In test/e2e_node/memory_eviction_test.go
> <https://github.com/kubernetes/kubernetes/pull/30654#discussion_r74853930>
> :
>
> > @@ -37,7 +37,9 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
> >  	f := framework.NewDefaultFramework("eviction-test")
> >
> >  	Context("When there is memory pressure", func() {
> > -		It("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
> > +		// Skip the test temporarily because of #30550.
> > +		// TODO(random-liu): Re-enable this test after #30550 is fixed.
> > +		PIt("It should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
>
> We should temporarily disable this to stop all of the flaky test emails
> related to this test, until @vishh <https://github.com/vishh> and I have
> a chance to discuss how to reliably force reclamation of memory.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30654/files/e3eea43cd8462328a74ef368af87a096d02935f7#r74853930>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGvIKIKTa5SjbDzbrg_IOrkzB-ISZoT4ks5qgPYHgaJpZM4Jk2ug>
> .
>